assigner/config.py

```diff
@@ -4,9 +4,11 @@
 
 from collections import UserDict
 
+
 class DuplicateUserError(Exception):
     pass
 
+
```

Comment:
@LinuxMercedes Ideas about why this file wasn't linted?

Reply:
It passes lint here -- maybe we're missing the rule for this problem?

Reply:
Maybe... I keep getting this feeling that pyflakes or pylint aren't getting some files due to how they scan via modules. Flake8 throws an absolute fit on this project.

Reply:
Oh, unrelated, we should switch to flake8 :/. Doing some reading, pyflakes was meant to be embedded in some other project.

Reply:
I threw an obvious lint problem in `config.py` and ran `pylint assigner` and it warned me about my problem, so I think we're good?

Reply:
Yeah :|